epam-promo/components/filters/FilterItemBody.tsx

```diff
@@ -11,30 +11,32 @@ export const FilterItemBody = (props: IFilterItemBodyProps<any>) => {
                 <FilterPickerBody
                     { ...props }
                     selectionMode="single"
-                    valueType="id"
+                    valueType={ props.valueType || "id" }
                 />
             );
         case "multiPicker":
             return (
                 <FilterPickerBody
                     { ...props }
                     selectionMode="multi"
-                    valueType="id"
+                    valueType={ props.valueType || "id" }
                 />
             );
         case "datePicker":
             return (
                 <FilterDataPickerBody
                     { ...props }
-                    format="DD/MM/YYYY"
+                    format={ props.format || "DD/MM/YYYY" }
                 />
             );
         case "rangeDatePicker":
             return (
                 <FilterRangeDatePickerBody
                     { ...props }
+                    format={ props.format || "DD/MM/YYYY" }
                     value={ props.value || { from: null, to: null } }
+                    changeIsOpen={ () => props.onClose() }
```

Comment:
props.onClose already passed via {...props} ahead, just use it into FilterRangeDatePickerBody